Doc/library/random.rst

```diff
@@ -166,7 +166,8 @@ Functions for sequences
    The *weights* or *cum_weights* can use any numeric type that interoperates
    with the :class:`float` values returned by :func:`random` (that includes
    integers, floats, and fractions but excludes decimals).  Weights are
-   assumed to be non-negative.
+   assumed to be non-negative.  A :exc:`ValueError` is raised if the
+   total of all weights isn't positive.
```

Comment:
To me this is a confusing explanation of an implementation trick.  Suggest instead something like:

"Behavior is undefined if any weight is negative.  A :exc:`ValueError` is raised if all weights are zero."